recipes/pytorch-cpu/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: pytorch-cpu
+  version: {{ version }}
+
+source:
+  # The tar source needs serious patching since it is trying to build
+  # all 3rd party modules
+  # url: https://github.com/pytorch/pytorch/archive/v{{ version }}.tar.gz
+  # fn: pytorch-v{{ version }}.tar.gz
+  # sha256: f360b7954e441bc5c4b36c49e711697a5c351de8b146c2982809385e3b3cfdcb
+  git_url: https://github.com/pytorch/pytorch.git
+  git_tag: v{{ version }}
+  patches:
+    - mac_osx_clock_getttime.patch  # [osx]
+
+# The build needs to set many environment variables
+# This is an easy way to do it cross platform
+{% set export = "export" %}  # [not win]
+{% set export = "set" %}     # [win]
+
+build:
+  number: 1000
```

Comment:
This should go back to 0.

Reply:
Ok, but this kind of stuff should really just be worked on in the feedstock. No builds are being set, and we can spend for ever making things "tidy".

Reply:
I mean, no actual package is going to get built once this feedstock gets created, so it is pretty safe to "accept now" and fixup later.

Especially since we might have to request more time on Travis anyway.

Reply:
I've never merged a recipe like this one before so I rather defer the merge to @conda-forge/core.

Reply:
I had to put it back to 1000 since we are now building with the gcc compilers on linux.